routes/mentor.go

```diff
@@ -3,8 +3,42 @@ package routes
 import (
 	"fmt"
 	"net/http"
+	"encoding/json"
+	"io/ioutil"
+	"github.com/jinzhu/gorm"
+	"kwoc20-backend/models"
 )
 
 func MentorReg(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "hello")
+
+	var mentor models.Mentor
+	body, _ := ioutil.ReadAll(r.Body)
+	err := json.Unmarshal(body, &mentor)
+	if err != nil {
+	  http.Error(w, err.Error(), 400)
+	  return 		
+	}
+
+	
+	db, err := gorm.Open("sqlite3", "kwoc.db")
+	if err != nil {
+		 http.Error(w, err.Error(), 500)
+		 return
+	}
+	mentor.db = db
```

Comment:
I dint get this?
directly db.Create is enuf right?

Reply:
I thought this can also be done. So, tried it once